deps/blas.mk

```diff
@@ -46,9 +46,11 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS := $(JFFLAGS)
+OPENBLAS_FFLAGS := $(JFFLAGS) $(USE_BLAS_FFLAGS)
 OPENBLAS_CFLAGS := -O2
 
+# OPENBLAS_FFLAGS += -frecursive -fno-optimize-sibling-calls -fdefault-integer-8
```

Comment:
This looks like something that slipped through. ;)

Reply:
Yup, thanks for pointing out!